### PR TITLE
Network Infrastructure

### PR DESCRIPTION
**Pull request by ar3l4**

### CODE REVIEW

File: oss-cloud-infrastructure/vpc-networking.yaml [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+      Tags:
+        - Key: Name
+          Value:
+            Fn::Sub: "#{AWS::StackName}-#{PublicSubnet.AvailabilityZone}"
+        - Key: Network
+          Value: Public
+  PublicSubnet:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: 'eu-west-1a'
```
Comment (mjovanovic) [inline oss-cloud-infrastructure/vpc-networking.yaml:49]: Can we use pseudo parameter `#{AWS::Region}` here and in other places in this file?

File: oss-cloud-infrastructure/serverless.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+service: tst-infrastructure
```
Comment (mjovanovic) [inline oss-cloud-infrastructure/serverless.yml:1]: If "tst" stands for test lets rename it. For example: "oss-infrastructure" ? The same applies to SSM parameters above.

File: oss-cloud-backend/serverless.yml [hunk 3]
```diff
@@ -5,6 +5,12 @@ provider:
   runtime: nodejs10.x
   region: eu-west-1
   stage: ${opt:stage, 'local'}
+  vpc:
+    securityGroupIds:
+      - ${ssm:/NETWORKING/tst-infrastructure-dev/LambdaSecurityGroup}
+    subnetIds:
+      - !Select [ 0, '${ssm:/NETWORKING/tst-infrastructure-dev/LambdaSubnets~split}' ]
+      - !Select [ 1, '${ssm:/NETWORKING/tst-infrastructure-dev/LambdaSubnets~split}' ]
```
Comment (mjovanovic) [inline oss-cloud-backend/serverless.yml:13]: Please update README with info about needed SSM parameters.

**Comment by ar3l4** [comment]
@mjovanovic I made changes that you listed earlier this morning. If there is anything else that I should change, please feel to point those things to me.

**Review by mjovanovic — APPROVED** [review]

**Pull request merged**